Python/PRP/pyCreatableConvert.cpp

```diff
@@ -842,7 +471,8 @@ PyObject* ICreate(plCreatable* pCre)
         case kSoundMsg: pySoundMsg_FromSoundMsg(plSoundMsg::Convert(pCre));
         default:
             // many messages are not implemented, make sure they are at least a plMessage
-            if (dynamic_cast<plMessage*>(pCre)) return pyMessage_FromMessage(plMessage::Convert(pCre));
+            if (dynamic_cast<plMessage*>(pCre))
+                return pyMessage_FromMessage(plMessage::Convert(pCre));
             return pyCreatable_FromCreatable(pCre);
```

Comment:
I can't help but wonder if there is some way to change all this crap into some kind of templatey goodness. It's pretty crazy that we have to have this kind of thing done by hand 😿